browse/static/js/paperswithcode.js

```diff
@@ -25,13 +25,38 @@
   });
 
   var icons = {
+    catalyzex: '<svg xmlns="https://www.catalyzex.com/assets/catalyzeX_logo-a747b368790c2d01e4abeb52322805149ea41b122784ace0770e896486aa9889.svg" class="pwc-icon pwc-icon-primary" viewBox="0 0 612 512" > <image href="https://www.catalyzex.com/assets/catalyzeX_logo-a747b368790c2d01e4abeb52322805149ea41b122784ace0770e896486aa9889.svg"></image> </svg>',
```

Comment:
this alphanumeric string based image url will change each time we redeploy our server. Can we instead fo something like the others have done with inline logo creation with path stroke etc?

Reply:
Found our `path d` data by opening our svg icon in the browser and doing inspect element and concatenating all the `d` strings

```suggestion
    catalyzex: '<svg xmlns="http://www.w3.org/2000/svg" class="cx-icon" viewBox="0 0 512 512"><path d="M405 3686 c-42 -18 -83 -69 -92 -114 -4 -20 -8 -482 -8 -1027 l0 -990 25 -44 c16 -28 39 -52 65 -65 38 -20 57 -21 433 -24 444 -3 487 1 538 52 18 18 37 50 43 71 7 25 11 154 11 343 l0 302 -165 0 -165 0 0 -240 0 -240 -225 0 -225 0 0 855 0 855 225 0 225 0 0 -225 0 -225 166 0 165 0 -3 308 c-3 289 -4 309 -24 342 -11 19 -38 45 -60 57 -39 23 -42 23 -469 22 -335 0 -437 -3 -460 -13z M1795 3686 c-16 -7 -38 -23 -48 -34 -47 -52 -46 -27 -47 -1262 0 -808 3 -1177 11 -1205 14 -50 63 -102 109 -115 19 -5 142 -10 273 -10 l238 0 -3 148 -3 147 -125 2 c-69 0 -135 1 -147 2 l-23 1 0 1025 0 1025 150 0 150 0 0 145 0 145 -252 0 c-188 -1 -261 -4 -283 -14z M3690 3555 l0 -145 155 0 155 0 0 -1025 0 -1025 -27 0 c-16 -1 -84 -2 -153 -3 l-125 -2 -3 -148 -3 -148 258 3 c296 3 309 7 351 88 l22 45 -2 1202 c-3 1196 -3 1202 -24 1229 -11 15 -33 37 -48 48 -26 20 -43 21 -292 24 l-264 3 0 -146z M2520 2883 c0 -5 70 -164 156 -356 l157 -347 -177 -374 c-97 -205 -176 -376 -176 -380 0 -3 77 -5 171 -4 l172 3 90 228 c49 125 93 227 97 227 4 0 47 -103 95 -230 l87 -230 174 0 c96 0 174 2 174 3 0 2 -79 172 -175 377 -96 206 -175 378 -175 382 0 8 303 678 317 701 2 4 -70 7 -161 7 l-164 0 -83 -210 c-45 -115 -85 -210 -89 -210 -4 0 -43 95 -86 210 l-79 210 -162 0 c-90 0 -163 -3 -163 -7z"></path></svg>',
```